packages/main/src/MultiInput.ts

```diff
@@ -13,81 +18,13 @@ import Input from "./Input.js";
 import MultiInputTemplate from "./generated/templates/MultiInputTemplate.lit.js";
 import styles from "./generated/themes/MultiInput.css.js";
 import Token from "./Token.js";
-import Tokenizer from "./Tokenizer.js";
+import Tokenizer, { ClipboardDataOperation, TokenizerTokenDeleteEventDetail } from "./Tokenizer.js";
```

Comment:
TokenizerTokenDeleteEventDetail should be imported with `import type`

Reply:
Done